src/tasks/platforms/minikube.ts

```diff
@@ -131,6 +131,50 @@ export class MinikubeTasks {
     ], { renderer: flags['listr-renderer'] as any })
   }
 
+  configureApiServer(flags: any): ReadonlyArray<Listr.ListrTask> {
+    return [
+      {
+        title: 'Copy certificate into Minikube',
+        task: async (ctx: any, task: any) => {
+          const args: string[] = []
+          args.push('cp')
+          args.push(ctx[OIDCContextKeys.CA_FILE])
+          args.push('/etc/ca-certificates/dex-ca.crt')
+
+          await execa('minikube', args, { timeout: 60000 })
+
+          task.title = `${task.title}...[OK]`
+        },
+      },
+      {
+        title: 'Configure Minikube API server',
+        task: async (ctx: any, task: any) => {
+          const args: string[] = []
+          args.push(`--extra-config=apiserver.oidc-issuer-url=${ctx[OIDCContextKeys.ISSUER_URL]}`)
+          args.push(`--extra-config=apiserver.oidc-client-id=${ctx[OIDCContextKeys.CLIENT_ID]}`)
+          args.push('--extra-config=apiserver.oidc-ca-file=/etc/ca-certificates/dex-ca.crt')
```

Comment:
Do we need to add this certificate to Che trusted CA bundle?

Reply:
@sparkoo mentioned that it is needed. I will rework.